packages/elements/src/components/typography/__styles__/index.ts

```diff
@@ -92,6 +93,10 @@ export const ElSubtitle = styled.h2`
   &.${elHasCenteredText} {
     text-align: center;
   }
+
+  &.${elHasCapitalisedText} {
```

Comment:
As discussed, can you add to all the Typeography classes, add to Storybook and changelog please